src/vaadin-overlay.html

```diff
@@ -715,7 +717,7 @@
 
         // Disable pointer events in other attached overlays
         OverlayElement.__attachedInstances.forEach(el => {
-          if (el !== this && !el.hasAttribute('opening') && !el.hasAttribute('closing')) {
+          if (el !== this) {
             el.shadowRoot.querySelector('[part="overlay"]').style.pointerEvents = 'none';
           }
```

Comment:
This was simplified a bit as this works like this now with the updated logic above. Now `pointer-events: none` might also be applied to `closing` or `opening` overlays here but that should be fine. Closing dialogs will remove it themselves after the closing animation finishes (in `finishClosing()`).